page-objects/pages/coursePages/CRSSCourseListPage.js

```diff
@@ -229,12 +233,18 @@ async function goToTasksOfTheCourse(coursename, section) {
 	await gotoTasksTab();
 }
 
-async function studentLogsInAndGoesToTasksOfTheCourse(username, password, coursename, section) {
-	await logoutPage.goToLogoutPage();
-	await startPage.performLogin(username, password);
-	await loginPage.firstLoginStudent(username, password);
-	await goToTasksOfTheCourse(coursename, section);
-}
+async function studentLogsInAndGoesToTasksOfTheCourse(username, password, coursename) {
+        await navigationTopPage.performLogout();
+        await startPage.performLogin(username, password);
+        await loginPage.firstLoginStudent(username, password);
+        await goToTasksOfTheCourse(coursename);
+};
+
+async function verifyCourseAndTopic(coursename, topicname, section) {
```

Comment:
this method verifyCourseAndTopic is the same like isTopicInCourseInSection , please remove verifyCourseAndTopic

Reply:
done, thanks